docs/running.rst

```diff
@@ -271,6 +271,51 @@ Typing ``python toil-sort-example.py --help`` will show the complete list of arg
 both Toil's and ones defined inside **toil-sort-example.py**.  A complete explanation of Toil's arguments can be found
 in :ref:`commandRef`.
 
+Environment Variable Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are several environment variables that affect the way Toil runs.
```

Comment:
Can you use Sphinx' definition lists? Search for `definition lists` in http://www.sphinx-doc.org/en/stable/rest.html.